xaitk_saliency/impls/gen_descriptor_sim_sal/similarity_scoring.py

```diff
@@ -8,20 +8,21 @@
 
 class SimilarityScoring (GenerateDescriptorSimilaritySaliency):
     """
-    This saliency implementation transforms proximity in feature
-    space into saliency heatmaps. This should
-    require a sequence of feature vectors of the query and
-    reference image, a number of feature vectors as predicted
-    on perturbed images, as well as the masks of the reference image
-    perturbations (as would be output from a
-    `PerturbImage` implementation.
+    This saliency implementation transforms proximity in feature space into
+    saliency heatmaps.
+    This should require a sequence of feature vectors of the query and reference
+    image, a number of feature vectors as predicted on perturbed version of the
+    query image, as well as the masks of the query image perturbations (as would
+    be output from a `PerturbImage` implementation.
 
     The perturbation masks used by the following implementation are
     expected to be of type integer. Masks containing values of type
     float are rounded to the nearest value and binarized
     with value 1 replacing values greater than or equal to half of
     the maximum value in mask after rounding while 0 replaces the rest.
 
+    The resulting saliency maps are relative to the query image.
```

Comment:
I think the wording here is a bit ambiguous, I suggest adding the same text you used below for the interface:

```suggestion
    The resulting saliency map is relative to the query image.
    As such, it denotes regions in the query image that make it more or less
    similar to the other image, called the reference image.
```